.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+Please make sure that these boxes are checked before closing your pull request.
```

Comment:
It would be nice to have a separate bar above this checklists so that the post comment can be easily distinguished.